dvc/command/experiments/utils.py

```diff
@@ -0,0 +1,32 @@
+def add_rev_selection_flags(
+    experiments_subcmd_parser, command: str, default: bool = True
+):
+    experiments_subcmd_parser.add_argument(
+        "-A",
+        "--all-commits",
+        action="store_true",
+        default=False,
+        help=(
+            f"{command} all experiments in the repository "
+            "(overrides `--rev`)."
+        ),
+    )
+    msg = f"{command} experiments derived from the specified Git commit"
+    if default:
+        msg += "(instead of `HEAD`)."
+    experiments_subcmd_parser.add_argument(
+        "--rev",
+        type=str,
+        default=None,
+        help=msg,
+        metavar="<commit>",
+    )
+    experiments_subcmd_parser.add_argument(
+        "-n",
+        "--num",
+        type=int,
+        default=1,
+        dest="num",
+        metavar="<num>",
+        help=f"{command} the last `num` commits from <commit>.",
```

Comment:
Should we indicate the default?
```suggestion
        help=f"{command} the last `num` commits from <commit> "
        "(HEAD by default).",
```
Unless this only works together with `--rev`

Reply:
p.s. in docs (https://dvc.org/doc/command-reference/exp/show#-n) se say "from HEAD". That's incorrect right? Incomplete rather

Reply:
Oh, sorry, there is a problem, here it is `commits` instead of `experiments`.  Because here what this `-n` flag do is actually extend the baseline `rev` to its history commits. And if the `-n` is -1 then it represents the whole history of a given revision (a tag, a branch, or a commit). And the experiments based on any of the commits from the history will be listed here.

Reply:
> what this -n flag do is actually extend the baseline rev to its history commits...
if the -n is -1 then it represents the whole history of a given revision

Got it, thanks. May leave another suggestion then... ⌛